### PR TITLE
4.18 version of the docs

### PR DESCRIPTION
**Pull request by DaanHoogland**

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by borisstoyanov** [comment]
@blueorangutan docbuild

**Review by borisstoyanov — APPROVED** [review]
LGTM, had a quick read, seems good

**Comment by andrijapanicsb** [comment]
broken formating
image

**Comment by andrijapanicsb** [comment]
image

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,85 +17,67 @@
...
+ custom guest OSses where added and mapping for those OSses where added,
+ problems may occur during upgrade. Part of the mitigation is to make sure
+ OSses that are newly mapped but should have already been in the guest_os
+ table are there. Make sure you apply thois before you start the new 4.18
+ management server.
 
-Note: This workaround removes the StorPool plugin support. StorPool
-users should not apply the workaround to continue using their Storpool
-storage.
+first check which you miss:
```
Comment (andrijapanicsb) [inline source/releasenotes/about.rst:69]:
```suggestion
first check which one you miss:
```

File: source/releasenotes/about.rst [hunk 2]
```diff
@@ -17,85 +17,67 @@
...
 
-This issue will be fixed in the upcoming CloudStack version 4.17.1.0.
+.. parsed-literal::
 
-[1] https://github.com/apache/cloudstack/issues/6455
+  SELECT * FROM cloud.guest_os WHERE display_name IN (´CentOS 8´, ´Debian GNU/Linux 10 (32-bit)´, ´Debian GNU/Linux 10 (64-bit)´, ´SUSE Linux Enterprise Server 15 (64-bit)´, ´Windows Server 2019 (64-bit)´)
```
Comment (andrijapanicsb) [inline source/releasenotes/about.rst:73]: I think we've discuss to make this explicitly as an EXAMPLE - i.e. to state here that below things are given just for example purposes. The mitigation makes me believe that anyone having an issue with upgrade should be able to fix it with some/all of the SQLs below - and this is not true - they might be missing other guest OS-es, not only those 4-5.

**Comment by andrijapanicsb** [comment]
Few small issues to be fixed (otherwise LGTM) - please comments.

File: source/releasenotes/about.rst (hunk 2 above)
Comment (DaanHoogland) [reply source/releasenotes/about.rst:73]: true, but only those five are used in creating new mappings, others are either created in this upgrade and than mapped, or not used in new mappings at all.

Comment (DaanHoogland) [reply source/releasenotes/about.rst:73]: I can word it more weakly, though.

File: source/releasenotes/about.rst [hunk 3]
```diff
@@ -17,85 +17,67 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a 4.17 LTS minor release with about 20 fixes
-since the 4.17.1.0 release.
+Apache CloudStack |release| is the initial 4.18 LTS release. It has over 300 fixes
+ and features since the 4.17.1.0 release.
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:21]:
```suggestion
and features since the 4.17.1.0 release.
```

File: source/releasenotes/about.rst [hunk 4]
```diff
@@ -17,85 +17,67 @@
...
 
-To mitigate the issue, a CloudStack administrator needs to do the
-following on version 4.17.0.0:
+Possible Issue with Guest OS IDs
+================================
 
-• On each management server: stop the CloudStack management service, remove the Storpool plugin jar on /usr/share/cloudstack-management/lib/cloud-plugin-storage-volume-storpool-4.17.0.0.jar and restart the Cloudstack management service
-• On each KVM host: stop the CloudStack agent service, remove the StorPool plugin jar on /usr/share/cloudstack-agent/lib/cloud-plugin-storage-volume-storpool-4.17.0.0.jar and restart the CloudStack agent service
+It has been noticed during upgrade testing that some environment, where
+ custom guest OSses where added and mapping for those OSses where added,
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:63]:
```suggestion
custom guest OSses where added and mapping for those OSses where added,
```

File: source/releasenotes/about.rst [hunk 5]
```diff
@@ -17,85 +17,67 @@
...
+ problems may occur during upgrade. Part of the mitigation is to make sure
+ OSses that are newly mapped but should have already been in the guest_os
+ table are there. Make sure you apply thois before you start the new 4.18
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:66]:
```suggestion
problems may occur during upgrade. Part of the mitigation is to make sure
OSses that are newly mapped but should have already been in the guest_os
table are there. Make sure you apply those before you start the new 4.18
```

File: source/releasenotes/about.rst [hunk 6]
```diff
@@ -17,85 +17,67 @@
...
+ management server.
```
Comment (DaanHoogland) [inline source/releasenotes/about.rst:67]:
```suggestion
management server.
```

File: source/releasenotes/about.rst (hunk 1 above)
Comment (DaanHoogland) [reply source/releasenotes/about.rst:69]:
```suggestion
first check which of the guest_os entries you miss:
```

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
@blueorangutan docbuild

**Pull request closed without merging**